packages/golang/build.sh

```diff
@@ -40,6 +40,7 @@ termux_step_make_install() {
 	cp pkg/include/* $TERMUX_GODIR/pkg/include/
 	cp -Rf lib/* $TERMUX_GODIR/lib
 	cp -Rf pkg/${TERMUX_GOLANG_DIRNAME}/* $TERMUX_GODIR/pkg/${TERMUX_GOLANG_DIRNAME}/
+        cp -Rf misc/ $TERMUX_GODIR/
```

Comment:
Please change the spaces to tab and bump the package

Reply:
May I know, how u put tab in GitHub editor in android chrome

Reply:
No idea, but looks like you figured it out :)